### PR TITLE
Fixing Tests and Cleaning up swift documentation

### PR DESCRIPTION
**Pull request by KyleLeneau**
I have been playing with this on the side and have stumbled across a number of issues with the documentation, tests and example. I have made this PR to clean up:

* Added `ConsoleKitExample` as a target for other to try running
* Fixed `ConsoleKitExample` to be more realistic of the current API
* Fixed `ConsoleKitTests` to pass correctly for a future bug fix
* Updated various documentation and examples in `ConsoleKit` target to match the current API

Side note, this project is pretty cool and could be a good example for mobile dev ops to use swift over Ruby or Python (which I am trying to do). There are still some missing gaps in this tool like some basic commands and flow. I think this project has promise of SPM library, Commandant, and could maybe align with other leading CLI libraries like Click on python.

Keep up the good work!

btw, is there any CI system setup or planned?

### CODE REVIEW

**Comment by calebkleveter** [comment]
All of the `vapor/` organization repos use the GitHub Workflows CI system. It looks slike everything passed.

I'll let this PR age for a bit longer before merging it, in case anyone else wants to review it.

File: Sources/ConsoleKit/Command/Command.swift [hunk 1]
```diff
@@ -37,14 +39,20 @@
...
+///     var input = CommandInput(arguments: CommandLine.arguments)
+///     var context = CommandContext(console: console, input: input)
+///
+///     try console.run(CoswayCommand(), with: context)
 ///
-/// Use `CommandConfig` to register commands and create a `CommandGroup`.
+/// Use `Commands` to register commands and create a `CommandGroup`.
 ///
-/// - note: You can also use `console.run(...)` to run a `CommandRunnable` manually.
+/// - note: You can also use `console.run(...)` to run a `AnyCommand` manually.
```
Comment (0xTim) [inline Sources/ConsoleKit/Command/Command.swift]: Nit - should be
> an `AnyCommand` manually

Comment (calebkleveter) [inline Sources/ConsoleKit/Command/Command.swift]:
```suggestion
/// - note: You can also use `console.run(...)` to run a an `AnyCommand` manually.
```

Comment (grundev) [reply Sources/ConsoleKit/Command/Command.swift]: (typo @calebkleveter ... choose _one_ 😄)

**Comment by KyleLeneau** [comment]
Thanks @tanner0101 @0xTim @calebkleveter for the support. I have pushed an updated change for the Nit =) and I have made a change to include `swift build` step during CI. 

This is the first time working with Github actions so let me know if you want it different, I am very comfortable with CI work in other environments (actually didn't know Github actions supported Swift, I am very intrigued now).

**Pull request merged**